packages/gatsby-transformer-hjson/src/__tests__/gatsby-node.js

```diff
@@ -1,6 +1,7 @@
 const Promise = require(`bluebird`)
 const os = require(`os`)
 const HJSON = require(`hjson`)
+const foo = require(`foobar`)
```

Comment:
damn linter :P

let's do just
```suggestion
require(`foobar`)
```
this shouldn't make eslint complain

alternatively just temporarly add `// eslint-disable-next-line` before this